### PR TITLE
Bug 1566924 - fixing broker client go resource types 

### PR DESCRIPTION
**Pull request by shawn-hurley**
Do not merge until broker-client-go can be merged.

### CODE REVIEW

File: crd/conversions.go [hunk 1]
```diff
@@ -115,199 +115,219 @@ func ConvertBundleToSpec(spec v1.BundleSpec, id string) (*apb.Spec, error) {
...
+		Status: v1alpha1.BundleInstanceStatus{
+			Bindings: bindings,
 		},
-		Parameters:   string(b),
-		BindingIDs:   bindingIDs,
-		DashboardURL: si.DashboardURL,
 	}, nil
 }
 
 // ConvertServiceInstanceToAPB will take a ServiceInstanceSpec its associated
```
Comment (jmrodri) [inline crd/conversions.go]: needs a comma after `ServiceInstanceSpec`

File: crd/conversions.go [hunk 2]
```diff
@@ -115,199 +115,219 @@ func ConvertBundleToSpec(spec v1.BundleSpec, id string) (*apb.Spec, error) {
...
-		return v1.StateInProgress
+		return v1alpha1.StateInProgress
 	case apb.StateSucceeded:
-		return v1.StateSucceeded
+		return v1alpha1.StateSucceeded
 	case apb.StateFailed:
-		return v1.StateFailed
+		return v1alpha1.StateFailed
 	}
 	// all cases should be coverd. we should never hit this code path.
```
Comment (jmrodri) [inline crd/conversions.go]: typo in comment: coverd -> covered

**Review by jmrodri — CHANGES_REQUESTED** [review]
Some comment changes (that weren't part of this PR but in the same code path, so we should fix them while we are here.

**Review by jmrodri — APPROVED** [review]

**Review by eriknelson — APPROVED** [review]
VISIACK, about what i thought things would look like.

**Pull request merged**